apps/studio/components/interfaces/Settings/Addons/IPv4SidePanel.tsx

```diff
@@ -172,11 +172,12 @@ const IPv4SidePanel = () => {
                       <p className="text-foreground-light">
                         Allow direct database connections via IPv4 address
                       </p>
-                      <div className="flex items-center space-x-1 mt-2">
+                      <div className="grid grid-cols-[max-content,1fr] space-x-1 mt-2">
                         <p className="text-foreground text-sm">{formatCurrency(option.price)}</p>
-                        <p className="text-foreground-light translate-y-[1px]">
+                        <p className="text-foreground-light translate-y-[2px]">
                           / month / database
                         </p>
+                        <span className="col-start-2 text-foreground-light">(billed hourly)</span>
```

Comment:
This is only true for Orb-based customers

Orb-based customers also have a note at the bottom of the modal

> There are no immediate charges. The addon is billed at the end of your billing cycle based on your usage and prorated to the hour.